src/Documents/Session/SessionOptions.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace RavenDB\Documents\Session;
+
+use RavenDB\Http\RequestExecutor;
+
+class SessionOptions
+{
+    private ?string $database = null;
+    private bool $noTracking = false; // true; // todo: [Marcin] check default value for this
```

Comment:
is something is bool in java then it gets false by default - so code is fine - you can remove TODO's